feat(config): add config example (#1)
M6Web/statuscake-extractor: README.md

 - Filesize_kb
 - Requests

## Configure

> To confiugure the project, copy config_dist.json on your config.json to add:

Reviewer: You can rename `config_dist.json` by `config_example.json` and write : 

```suggestion
To confiugure the project, rename `config_example.json` at the root to `config.json`. You can add:
```

> To confiugure the project, copy `config_example.json` on your `config.json` to add:

Reviewer 2: ```suggestion
To configure the project, copy `config_example.json` on your `config.json` to add:
```
